dev/breeze/src/airflow_breeze/global_constants.py

```diff
@@ -777,19 +777,19 @@ def generate_provider_dependencies_if_needed():
 PROVIDERS_COMPATIBILITY_TESTS_MATRIX: list[dict[str, str | list[str]]] = [
     {
         "python-version": "3.10",
-        "airflow-version": "2.10.5",
+        "airflow-version": "2.11.0",
```

Comment:
The min Airflow version supported by providers is still 2.10. I think we should keep 2.10 in tests and simply add a new one for 3.1.0

Reply:
You are right. I just undid the change. Will add 3.1.0 to this in a separate PR anyway as some of the Provider compat tests are failing

Reply:
https://github.com/apache/airflow/pull/56118